docs/guides/js-waku/opchan-react.md

```diff
@@ -0,0 +1,537 @@
+---
+title: Build Forum UIs with @opchan/react
+sidebar_label: Build Forum UIs with OpChan
+description: Ship decentralized forum experiences over Waku using the OpChan React SDK.
+---
+
+`@opchan/react` adds a forum-focused state layer on top of `@opchan/core`, Wagmi, and the Waku network.
+
+## Why OpChan on Waku?
+
+- **Forum-first primitives** – cells, posts, comments, votes, bookmarks, and moderation helpers.
+- **Identity blending** – anonymous sessions, wallet accounts, ENS verified owners, and call signs share one cache.
+- **Permission reasoning** – UI gates and string reasons with zero custom role logic.
+- **Delegated signing** – short-lived browser keys prevent repetitive wallet prompts while keeping signatures verifiable.
+- **Local-first sync** – IndexedDB hydrates instantly while Waku relays stream live updates in the background.
+
+## Prerequisites
+
+- React 18+ with a modern build tool (Vite examples below).
+- Node.js 18+ and a package manager (npm or yarn).
+- A WalletConnect / Reown project id (`VITE_REOWN_PROJECT_ID`) so users can connect wallets.
+- Basic familiarity with [Waku content topics](/learn/concepts/content-topics) and message reliability concepts.
+
+## Quick Links
+
+- [Install the SDKs](#install-the-sdks)
+- [Bootstrap the provider](#1-bootstrap-the-provider)
+- [Hook surface overview](#2-understand-the-hook-surface)
+- [Common UI patterns](#3-compose-ux-patterns)
+- [Authentication flows](#4-authentication-flows)
+- [Reference app (collapsible)](#6-reference-layout)
+- [Troubleshooting](#8-troubleshooting)
+
+## Install the SDKs
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+<Tabs groupId="pkg-manager">
+<TabItem value="npm" label="npm">
+
+```bash
+npm install @opchan/react react react-dom buffer
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @opchan/react react react-dom buffer
+```
+
+</TabItem>
+</Tabs>
+
+For Vite + TypeScript projects also install `@vitejs/plugin-react`, `typescript`, and `@types/react`.
+
+## 1. Bootstrap the provider
+
+Add the Buffer polyfill (still required by many crypto libraries) and wrap your tree with `OpChanProvider`. The provider internally wires Wagmi, React Query, and the OpChan client to the Waku relay layer. Configure it with the content topic you want to publish under plus the reliable channel id you use for the [Waku Reliable Message Channel](https://specs.vac.dev/specs/waku/waku-reliable-message-delivery/).
+
+```tsx title="src/main.tsx"
+import { createRoot } from 'react-dom/client';
+import { OpChanProvider } from '@opchan/react';
+import { Buffer } from 'buffer';
+import App from './App';
+
+if (!(window as any).Buffer) {
+  (window as any).Buffer = Buffer;
+}
+
+const config = {
+  wakuConfig: {
+    contentTopic: '/opchan/1/messages/proto',
+    reliableChannelId: 'opchan-messages',
```

Comment:
Maybe add a comment that by using this value, they will get the data from the deployed https://opchan.app/ instance; they should change both values if they want different their own data silo.